central/graphql/resolvers/test_utils.go

```diff
@@ -573,3 +657,41 @@ func getNodeVulnerabilityResolver(ctx context.Context, t *testing.T, resolver *R
 	require.Equal(t, vulnID, vuln.Id(ctx))
 	return vuln
 }
+
+func getImageCVEDatastore(ctx context.Context, db *pgxpool.Pool, gormDB *gorm.DB) (imageCVEDataStore.DataStore, error) {
+	imageCVEStore := imageCVEPostgres.CreateTableAndNewStore(ctx, db, gormDB)
+	imageCVEIndexer := imageCVEPostgres.NewIndexer(db)
+	imageCVESearcher := imageCVESearch.New(imageCVEStore, imageCVEIndexer)
+	imageCVEDatastore, err := imageCVEDataStore.New(imageCVEStore, imageCVEIndexer, imageCVESearcher, dackboxConcurrency.NewKeyFence())
+	return imageCVEDatastore, err
+}
+
+func getImageDatastore(ctx context.Context, db *pgxpool.Pool, gormDB *gorm.DB, risks riskDataStore.DataStore) imageDatastore.DataStore {
+	imageStore := imagePostgres.CreateTableAndNewStore(ctx, db, gormDB, false)
+	return imageDatastore.NewWithPostgres(imageStore, imagePostgres.NewIndexer(db), risks, ranking.NewRanker(), ranking.NewRanker())
+}
+
+func getImageComponentDatastore(ctx context.Context, db *pgxpool.Pool, gormDB *gorm.DB, risks riskDataStore.DataStore) imageComponentDataStore.DataStore {
+	imageCompStore := imageComponentPostgres.CreateTableAndNewStore(ctx, db, gormDB)
+	imageCompIndexer := imageComponentPostgres.NewIndexer(db)
+	imageCompSearcher := imageComponentSearch.NewV2(imageCompStore, imageCompIndexer)
+	return imageComponentDataStore.New(nil, imageCompStore, imageCompIndexer, imageCompSearcher, risks, ranking.NewRanker())
+}
+func getImageCVEEdgeDatastore(ctx context.Context, db *pgxpool.Pool, gormDB *gorm.DB) imageCVEEdgeDataStore.DataStore {
+	imageCveEdgeStore := imageCVEEdgePostgres.CreateTableAndNewStore(ctx, db, gormDB)
+	imageCveEdgeIndexer := imageCVEEdgePostgres.NewIndexer(db)
+	imageCveEdgeSearcher := imageCVEEdgeSearch.NewV2(imageCveEdgeStore, imageCveEdgeIndexer)
+	return imageCVEEdgeDataStore.New(nil, imageCveEdgeStore, imageCveEdgeSearcher)
+}
+
+func getImageComponentCVEEdgeDatastore(ctx context.Context, db *pgxpool.Pool, gormDB *gorm.DB) componentCVEEdgeDataStore.DataStore {
```

Comment:
Refer to `central/graphql/resolvers/testutils.go`

Reply:
I can work on collapsing the `testutils.go` file with the `test_utils.go` file in some future refactor